miqcli/constants.py

```diff
@@ -63,8 +63,20 @@
     'enable_ssl_verify': False
 }
 
-SUPPORTED_AUTOMATE_REQUESTS = ['generic', 'gen_floating_ip',
-                               'release_floating_ip']
+
+#: automation requests options
```

Comment:
I like what you did here with this approach. This can also be applied to `provision_requests.py` where it is conditional based on creating the payload for a certain provider.